src/platform-apps/channels/channel-view.test.tsx

```diff
@@ -29,15 +39,35 @@ describe('ChannelView', () => {
     return shallow(<ChannelView {...allProps} />);
   };
 
-  it('renders channel name', () => {
+  const subjectWithContext = (props: any = {}, context: any = {}) => {
+    const allProps = {
+      name: '',
+      messages: [],
+      user: null,
+      countNewMessages: 0,
+      ...props,
+    };
+
+    const store = createStore(reducer);
+
+    return mount(
+      <Provider store={store}>
```

Comment:
i think this is a sign that something is wrong. we shouldn't need to provide a store within our tests. what is the reason for adding this? in the off chance that we *do* need the store, we should make sure we're adding the global store, not creating one like this, as the data structure will be different in this case than it is for the global store.